staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/fairqueuing_test.go

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
I think we can retire this file and test only through the QueueSetFactory and QueueSet interfaces, as fq_test.go does.

Reply:
Done.

Reply:
Done.  Removed this file and renamed fq_test.go -> fairqueuing_test.go